shippable.yml

```diff
@@ -14,11 +14,11 @@ build:
     - apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 23E7166788B63E1E 6A030B21BA07F4FB 4B8EC3BAABDC4346 EB3E94ADBE1229CF 960B2B2623A0BD5D 6B05F25D762E3157
     #- rm /etc/apt/sources.list.d/jonathonf-ubuntu-backports-xenial.list 
     - rm /etc/apt/sources.list.d/basho_riak.list
-    # install gradle 5.6.4
-    - wget https://services.gradle.org/distributions/gradle-5.6.4-bin.zip
+    # install Gradle 7.6
+    - wget https://services.gradle.org/distributions/gradle-7.6-bin.zip
     - sudo mkdir /opt/gradle
-    - unzip -d /opt/gradle gradle-5.6.4-bin.zip
-    - export PATH=/opt/gradle/gradle-5.6.4/bin:$PATH 
+    - unzip -d /opt/gradle gradle-7.6-bin.zip
+    - export PATH=/opt/gradle/gradle-7.6/bin:$PATH
     - gradle -v
     - java -version
     # ensure all modifications created by 'mature' generators are in the git repo
```

Comment:
FYI. thanks for updating it but we no longer use Shippable CI. 

I'll file another PR to clean up this file later.